public/index.html

```diff
@@ -21,6 +21,8 @@
     -->
     <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.0.13/css/all.css" integrity="sha384-DNOHZ68U8hZfKXOrtjWvjxusGo9WQnrNx2sqG0tfsghAvtVlRW3tvkXWZh58N9jp" crossorigin="anonymous">
 
+    <script type="text/javascript" src="https://maps.googleapis.com/maps/api/js?key=AIzaSyC63iYNVwNtcZ5e_6-fr2X5kjlSANRPXXc&libraries=places"></script>
```

Comment:
I think there may be a way to use the current gapi initialization to initialize the places apis, but I'm not sure. I'll take a look.